pyperf/_manager.py

```diff
@@ -102,10 +105,12 @@ def spawn_worker(self, calibrate_loops, calibrate_warmups):
                 proc = subprocess.Popen(cmd, env=env, **kw)
 
             with popen_killer(proc):
-                with rpipe.open_text() as rfile:
-                    bench_json = rfile.read()
-
-                exitcode = proc.wait()
+                try:
+                    bench_json = rpipe.read_text()
+                    exitcode = proc.wait()
```

Comment:
Can you add an EXIT_TIMEOUT=60 constant (at the module level) and use it for `wait(timeout=EXIT_TIMEOUT)`? A process *should* exit immediately once the pipe is closed, but it's better to be safe here.